From: Gordon Ball <gordon@chronitis.net>
Date: Wed, 23 Aug 2017 21:17:15 +0000
Subject: Disable sphinxcontrib_github_alt in documentation

This extension is unavailable and causes the documentation build to
fail.
---
 docs/conf.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 1a432bf..e1d6814 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -44,7 +44,6 @@ if os.environ.get('READTHEDOCS', ''):
 # ones.
 extensions = [
     'sphinx.ext.intersphinx',
-    'sphinxcontrib_github_alt',
 ]
 
 github_project_url = "https://github.com/jupyter/jupyter_console"
